NitroxClient/Debuggers/SceneDebugger.cs

```diff
@@ -115,6 +170,39 @@ private void RenderTabGameObject()
             }
         }
 
+        private void RenderTabMonoBehaviour()
+        {
+            using (new GUILayout.VerticalScope("Box"))
+            {
+                if (selectedMonoBehaviour)
+                {
+                    using (GUILayout.ScrollViewScope scroll = new GUILayout.ScrollViewScope(monoBehaviourScrollPos))
+                    {
+                        monoBehaviourScrollPos = scroll.scrollPosition;
+                        using (new GUILayout.HorizontalScope())
+                        {
+                            GUILayout.Label($"MonoBehaviour: {selectedMonoBehaviour.GetType().Name}", "header");
+                            editMode = GUILayout.Toggle(editMode, "Edit Mode");
+                            sendToServer = GUILayout.Toggle(sendToServer, "Send to server");
```

Comment:
Also hide this on `Multiplayer.Main == null`. Because this happens on multiple places maybe just create a function for it: `RenderSendToServerToggle()`.